client/extensions/woocommerce/lib/order-status/index.js

```diff
@@ -1,3 +1,4 @@
+/** @format */
```

Comment:
Whats this?

Reply:
It's from [prettier](https://github.com/Automattic/calypso-prettier), which I used to auto-format these files. This comment triggers [a pre-commit hook](https://github.com/Automattic/wp-calypso/blob/master/bin/pre-commit-hook.js#L48) so all added code is also auto-formatted.